src/core/tests/arbitraries.ts

```diff
@@ -451,6 +459,68 @@ export class Arbitraries {
         })
       )
 
+  ira = (): Arbitrary<types.Ira> =>
+    fc
+      .tuple(
+        words, //payer
```

Comment:
you can just make this an arbitrary string, no reason to restrict to alpha chars.